ImageDownloader.py

```diff
@@ -2,11 +2,16 @@
 
 import urllib
 import os
+from urlparse import urlparse
+from os.path import splitext,basename
+from multiprocessing import Pool,TimeoutError
 
 class ImagesDownload(object):
 
-	def __init__(self, fileName = None):
+	def __init__(self, pool_size=5,timeout=10,fileName = None):

```

Comment:
Could you please make changes to ImageDownloader.py file as you suggested.
```
images = ImagesDownload(fileName="ImageCollection.txt")
```
Why did I not think of that!? ;)